src/main/java/com/minecolonies/colony/buildings/AbstractBuilding.java

```diff
@@ -178,10 +192,10 @@ public static AbstractBuilding createFromNBT(Colony colony, NBTTagCompound compo
             {
                 BlockPos pos = BlockPosUtil.readFromNBT(compound, TAG_LOCATION);
                 Constructor<?> constructor = oclass.getDeclaredConstructor(Colony.class, BlockPos.class);
-                building = (AbstractBuilding)constructor.newInstance(colony, pos);
+                building = (AbstractBuilding) constructor.newInstance(colony, pos);
             }
-        }
-        catch (NoSuchMethodException | InstantiationException | InvocationTargetException | IllegalAccessException exception)
+        } catch (NoSuchMethodException | InstantiationException | InvocationTargetException
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this "catch" keyword to a new dedicated line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3ARightCurlyBraceDifferentLineAsNextBlockCheck)